dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathEntry.java

```diff
@@ -48,22 +48,18 @@
 
   /** An entry for a Maven artifact. */
   public ClassPathEntry(Artifact artifact) {
-    checkNotNull(artifact.getFile());
+    this(artifact.getFile().toPath());
```

Comment:
This defies line 39 "Either jar or artifact is non-null".

Reply:
removed